msg/versioned/ArmingCheckReply.msg

```diff
@@ -1,34 +1,38 @@
+# Arming check reply.
+#
+# Response to an ArmingCheckRequest message.
+
 uint32 MESSAGE_VERSION = 0
 
-uint64 timestamp # time since system start (microseconds)
+uint64 timestamp # Time since system start [us]
 
-uint8 request_id
-uint8 registration_id
+uint8 request_id # Id of ArmingCheckRequest for which this is a response.
+uint8 registration_id # Id of component emitting this response.
```

Comment:
As above, what is a "registration". I've referred to it as a component, but it looks like it might be a flight mode.